tasks/setup-Debian.yml

```diff
@@ -6,7 +6,7 @@
 
 - name: Add Logstash repository.
   apt_repository:
-    repo: 'deb http://packages.elasticsearch.org/logstash/2.3/debian stable main'
+    repo: 'deb "{{logstash_debian_repo_version}}" stable main'
```

Comment:
I'd rather do it the same way I do the Kibana role, but with a `logstash_version` variable—assuming that would work here: https://github.com/geerlingguy/ansible-role-kibana/blob/master/tasks/setup-Debian.yml#L12